2017/WG-09-12.md

```diff
@@ -0,0 +1,71 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the September 12th video call of WebAssembly's Working Group
+
+- **Host**: Google Hangouts
+- **Dates**: Tuesday September 12th, 2017
+- **Times**: 9:00am–10:00am Pacific Time
+- **Location**: *same Google Hangouts as previous CG video meetings*
+- **Contact**:
+    - Name: Brad Nelson
+    - Email: bradnelson@google.com
+
+### Registration
+
+None required if you've attended before. Email Brad Nelson to sign up if it's
+your first time. The meeting is open to WG members only (we'll likely broaden
+this in future meetings).
+
+## Logistics
+
+The meeting will be a Google Hangouts call.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking.
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Openness (Brad Nelson)
+       1. Discussion on how open the WG process should be.
+       1. POLL: We comfortable with opening WG meetings to all CG members (for observation).
+       1. POLL: We comfortable opening WG meetings to the public (for observation).
+    1. Scope of v.1 specification (Brad Nelson)
+       1. Discussion on which things should be included in the initial
+          specific, possibilities include:
+          1. [Binary format spec](https://github.com/WebAssembly/spec)
+             1. Core format
+             1. Reference interpreter
+             1. Test suite
+          1. [JavaScript embedding](https://github.com/WebAssembly/design/blob/master/JS.md)
+          1. [Web embedding](https://github.com/WebAssembly/design/blob/master/Web.md)
+       1. POLL: We should limit v.1 to a binary format specification only.
+    1. Adoption of CG specification (Andreas Rossberg)
+       1. Discussion of state of the binary specification.
+       1. POLL: We should officially tag and adopt a revision of the spec as
+          the WG input.
+       1. POLL: We should declare this version the "First Public Working Draft".
```

Comment:
Is this poll formally adopting it, or saying that we should perform work towards adopting it?

Reply:
Meant the former, though not sure how the poll will go (and likely we might not even get to that bullet). Also planning to seek some guidance from Eric + others.

Reply:
I don't think we can formally adopt it as-is, since we haven't been told that it was done, or that we should review it until just now.